ext/storage_sources/azure_store/test/test_azure_connection.cpp

```diff
@@ -31,4 +31,39 @@
 
 #include "azure_connection.h"
 
-TEST_CASE("testing class azure_connection", "azure_connection") {}
+TEST_CASE("testing class azure_connection", "azure_connection")
+{
+
+    std::vector<std::string> objects;
+    azure_connection pfx_test = azure_connection("myblobcontainer1", "pfx_test_");
+
+    // There is nothing in the container so there should be 0 objects
```

Comment:
Does the state of the container affect your unit testing? I don't think you need to check for an empty container. An assertion that put_object worked should be enough to set up your testing environment

Reply:
yup removed this test case https://github.com/wiredtiger/wiredtiger/pull/8738/commits/62f7e4766feef2010b903baa8e78a7234e1274da